pkg/runtime/runtime.go

```diff
@@ -1768,12 +1768,14 @@ func (a *DaprRuntime) initActors() error {
 	a.actorStateStoreLock.Lock()
 	defer a.actorStateStoreLock.Unlock()
 	if a.actorStateStoreName == "" {
-		return errors.New("no actor state store defined")
+		log.Warn("actors: state store is not configured - this is okay for clients but services with hosted actors will fail to initialize!")
```

Comment:
Will this introduce a warning to all instances that don't use actors? If it's OK in some scenarios we shouldn't say anything here.

Reply:
I could change it to info level -- we warn if the app has hosted actors but no actor state store. I think it's maybe worth letting people know informationally, but we can also probably get rid of it if it's overly verbose. I personally prefer more logs to sift through than less context but will go with whatever our usual policy / behavior is.

Reply:
I'm also OK with slightly more verbose logging, especially at startup. I just don't think we should warn someone if their behavior is expected and rational. I'm fine with moving it to an info, it's a log message that happens once during init and it's a piece of information they may want. Warning, to me, means it's something they should go fix but not critically.

Reply:
Swapped to Info in 32dbce326945caa3f2b7a4ee6c4d9a6704cc51a0